app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+
+    if len(command) != 3:
+        raise ValueError(
+            f"unexpected number of values (expected 3, got {len(command)})"
+        )
+    if command[0] != "mv":
+        raise ValueError(f"{command[0]}: command not found")
+
+    old_file_name = command[1]
+    new_file_name = command[2]
+
+    if not os.path.exists(old_file_name):
```

Comment:
you can rather wrap your lines 31 and 32 in try except block which will cover more issues that can happen when working with file